tutorial/chapter/3/index.md

```diff
@@ -173,11 +184,12 @@ fn foo(x) x + 1
 This is useful for functions that need to refer to themselves, like:
 
 ```squiggle
-fn forever(f)
-    if done
-    then undefined
+def forever(f)
```

Comment:
In the JS I'm used to seeing, functions are usually defined with a function statement, which `def` is supposed to mirror. You could use anonymous functions, but it makes debugging harder. I could infer the name based on the assignment, but it seems excessively magical. Most importantly, `let foo = fn(x) do` feels clunky compared to `def foo(x)` to me.